egs2/l3das22/enh1/local/metric.py

```diff
@@ -0,0 +1,139 @@
+# The implementation of the metric for L3DAS22 in
+# Guizzo. et al. "L3DAS22 Challenge: Learning 3D Audio
+# Sources in a Real Office Environment"
+# The code is based on:
+# https://github.com/l3das/L3DAS22/blob/main/metrics.py
+
+
+import argparse
+import os
+import sys
+import warnings
+
+import numpy as np
+import soundfile as sf
+import torchaudio
+import torch
+from tqdm import tqdm
+
+import jiwer
+import numpy as np
+from pystoi import stoi
+import transformers
```

Comment:
Is it OK?
I think we need to install something.

Reply:
Yes, we need to install jiwer (WER), pystoi (STOI), and transformers (ASR).
I can add an installation part to the metric.sh.